packages/common/src/ui/layout/tables/components/Header/Header.tsx

```diff
@@ -82,18 +100,7 @@ export const HeaderCell = <T extends RecordWithId>({
         placement="bottom"
         style={{ whiteSpace: 'pre-line' }}
       >
-        {sortable ? (
-          <TableSortLabel
-            hideSortIcon={false}
-            active={isSorted}
-            direction={direction}
-            IconComponent={SortDescIcon}
-          >
-            <Header column={column} />
-          </TableSortLabel>
-        ) : (
-          <Header column={column} />
-        )}
+        {HeaderLabel}
```

Comment:
Nice. I originally found this because I was thinking about adding them to the edit modal column headers to explain things like what 'available' and 'on hold' meant.

Also now that i'm thinking about tooltips... :)

Maybe a good idea to wrap the `BasicCell` (and maybe some other cells) in a tooltip which displays if the text has overflowed (and is using ellipsis for text overflow) to show the full value. For example the `locationName` doesn't have too much room I could imagine that could be quite annoying.

Reply:
good idea! will merge this one, to get rid of those warnings and have a look at a `BasicCell` overflow tooltip